hist/hist/inc/TH2.h

```diff
@@ -57,11 +57,20 @@ class TH2 : public TH1 {
    Int_t    Fill(Double_t); //MayNotUse
    Int_t    Fill(const char*, Double_t) { return Fill(0);}  //MayNotUse
 
+   virtual Double_t Interpolate(Double_t x) const; // may not use
+
 private:
 
    TH2(const TH2&);
    TH2& operator=(const TH2&); // Not implemented
 
+   // make private methods which have a TH1 signature and should not
+   // be called for a TH2
+   using TH1::GetBinErrorLow;
+   using TH1::GetBinErrorUp;
+   using TH1::Integral;
+   using TH1::IntegralAndError;
+
```

Comment:
While trying to integrate this changes
https://github.com/cms-sw/cmsdist/pull/5253
the rebuild of yoda failed as yoda uses TH2::GetBinErrorUp
in here:
```
inline Scatter3D toScatter3D(const TH2& th2, bool scalebyarea=true) {
    Scatter3D rtn;
    for (int ix = 1; ix <= th2.GetNbinsX(); ++ix) {
      for (int iy = 1; iy <= th2.GetNbinsY(); ++iy) {
        const TAxis& axx = *th2.GetXaxis();
        const double x = axx.GetBinCenter(ix);
        const double exminus = x - axx.GetBinLowEdge(ix);
        const double explus = axx.GetBinUpEdge(ix) - x;
        const double xwidth = axx.GetBinWidth(ix);
        //
        const TAxis& axy = *th2.GetYaxis();
        const double y = axy.GetBinCenter(iy);
        const double eyminus = y - axy.GetBinLowEdge(iy);
        const double eyplus = axy.GetBinUpEdge(iy) - y;
        const double ywidth = axy.GetBinWidth(iy);
        //
        const int ixy = th2.GetBin(ix, iy);
        const double area = xwidth*ywidth;
        const double val = th2.GetBinContent(ixy) / (scalebyarea ? area : 1);
        const double evalminus = th2.GetBinErrorLow(ixy) / (scalebyarea ? area : 1);
        const double evalplus = th2.GetBinErrorUp(ixy) / (scalebyarea ? area : 1);
        rtn.addPoint(x, y, val,
                     exminus, explus,
                     eyminus, eyplus,
                     evalminus, evalplus);
      }
    }
```

If I'm about to patch yoda, what should I use instead of GetBinErrorUp and GetBinErrorLow ?